MvvmCross/Binding/iOS/Target/MvxUITextViewTextTargetBinding.cs

```diff
@@ -46,7 +46,16 @@ public override void SubscribeToEvents()
                 return;
             }
 
-            target.Changed += this.EditTextOnChanged;
+			var textStorage = target.LayoutManager?.TextStorage;
+
+			if (textStorage == null)
+			{ 
+			    MvxBindingTrace.Trace(MvxTraceLevel.Error,
+						  "Error - NSTextStorage of UITextView is null in MvxUITextViewTextTargetBinding");
+				return;
+			}
+
+            textStorage.DidProcessEditing += this.EditTextOnChanged;
```

Comment:
This event is marked Deprecated in the Apple documentation: https://developer.apple.com/reference/uikit/nstextstoragedelegate